src/subgraph/client.ts

```diff
@@ -12,7 +12,7 @@ export interface SubgraphClient {
   getDomainsByName(name: string): Promise<Domain[]>;
   getDomainsByOwner(owner: string): Promise<Domain[]>;
   getSubdomainsById(domainId: string): Promise<Domain[]>;
-  getRecentSubdomainsById(domainId: string): Promise<Domain[]>;
+  getMostRecentSubdomainsById(domainId: string, count: number | undefined): Promise<Domain[]>;
```

Comment:
A really handy shorthand for something like `count: number | undefined` is to add a `Maybe` type. 

The declaration looks like this:
`type Maybe<T> = T | undefined`

then you can just do `count: Maybe<number>`